server/src/parser/symbols.ts

```diff
@@ -8,12 +8,17 @@ import { ASTVisitor } from './visitor';
 export enum CodeSymbolType {
```

Comment:
Instead of having a local/global version of each type, it seems like it'd be better to have the enum just be function/variable/table/array and then a separate boolean for local vs. global.

Reply:
Yeah I agree